libs/designsystem/src/lib/scss/base/typography.spec.ts

```diff
@@ -238,4 +242,43 @@ describe('Typography', () => {
       });
     }
   });
+
+  describe(`anchor tag`, () => {
+    describe(`by default`, () => {
+      beforeEach(() => {
+        const fixture = `<a id="fixture">Text</a>`;
```

Comment:
This should probably have a `href` attribute. Otherwise it's just an indicated part of the document (an anchor) and not a hyperlink.